api-reference/10 UI Widgets/dxDataGrid/4 Events/fileSaving.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: dxDataGrid.fileSaving
 type: eventType
-deprecated: 
+deprecated: Since v20.1, we recommend using [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```

Comment:
Can we use an include so as not to copy-paste the message?

Reply:
It does not seem to work as intended